tests/platform_tests/test_memory_exhaustion.py

```diff
@@ -24,7 +24,8 @@ class TestMemoryExhaustion:
     """
 
     @pytest.fixture(autouse=True)
-    def tearDown(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, pdu_controller):
+    def tearDown(self, duthosts, enum_rand_one_per_hwsku_hostname,
+                 localhost, pdu_controller,):
```

Comment:
remove extra comma at end

Reply:
also fix the line indentation

Reply:
removed the comma.
Indentation is correct I believe: https://github.com/sonic-net/sonic-mgmt/blob/9d6c0de664f9878b1b2b4282cab95771a7ed01b4/tests/platform_tests/api/test_sfp.py#L37